rpcd/playbooks/roles/rpc_support/tasks/main.yml

```diff
@@ -95,3 +95,4 @@
 
 - include: configure_unattended_upgrades.yml
 
+- include: create_default_flavors.yml
```

Comment:
Instead of using the ``delegate_to`` keyword to distribute tasks, it would be better to conditionally include this playbook by modifying this ``- include`` to look like this:
```
- include: create_default_flavors.yml
  when: "{{ inventory_hostname in groups['utility'] }}"
```